include/librealsense2/h/rs_device.h

```diff
@@ -146,6 +146,53 @@ void rs2_connect_tm2_controller(const rs2_device* device, const unsigned char* m
 */
 void rs2_disconnect_tm2_controller(const rs2_device* device, int id, rs2_error** error);
 
+/**
+* Set tm2 camera intrinsics
+* \param[in] device       tm2 device
+* \param[in] sensor_id    specify fisheye lens
+* \param[in] intr         intrinsics value to be written to the device
+* \param[out] error       If non-null, receives any error that occurs during this call, otherwise, errors are ignored
+* \return 0 if the operation is completed successfully.
+*/
+void rs2_set_tm2_intrinsics(const rs2_device* device, int sensor_id, const rs2_intrinsics* intr, rs2_error** error);
+
+/**
+   * Set tm2 camera extrinsics
+   * \param[in] device          tm2 device
+   * \param[in] stream_type     sensor stream type
+   * \param[in] stream_index    sensor stream index
+   * \param[in] extr            extrinsics value to be written to the device
+   * \param[out] error          If non-null, receives any error that occurs during this call, otherwise, errors are ignored
+   * \return 0 if the operation is completed successfully.
+   */
+void rs2_set_tm2_extrinsics(const rs2_device* device, rs2_stream stream_type, int stream_index, const rs2_extrinsics* extr, rs2_error** error);
```

Comment:
What are the constrains of the API - can it be applied at any time or during idle state only?
Is it a persistent change or will it reset to internally-stored values after reset?

Reply:
@ev-mp as far as I know, those set functions will modify the virtual table of the device. So it won't affect run time result until the device being reset. 
Even said that, we don't recommend the change those parameters during run-time.
@cchen6 @bfulkers-i, any comment?

Reply:
They will take effect once you start device/play which will load calibration. It doesn't make sense to call these functions during play.